src/connectors/jamstash.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+Connector.playerSelector = '#player';
+
+Connector.artistSelector = '#songdetails .artist';
+
+Connector.trackSelector = '#songdetails .song';
+
+Connector.albumSelector = '#songdetails .album';
+
+Connector.isPlaying = () => {
+	return $('#player .PlayTrack').css('display') === 'none';
```

Comment:
This could be simplified by using `playButtonSelector` property:
```js
Connector.playButtonSelector = '#player .PlayTrack';
```

Reply:
fixed